src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/form/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import {getDescriptionTemplate, getGoalTemplate, getHeaderTemplate, getTitleTemplate} from '../templates';
+import getWindowData from '../utilities/getWindowData';
+import {GoalType} from '@givewp/blocks/form/app/templates/layouts/Goal';
+
+const {form} = getWindowData();
+
+const HeaderTemplate = getHeaderTemplate();
+const TitleTemplate = getTitleTemplate();
+const DescriptionTemplate = getDescriptionTemplate();
+const GoalTemplate = getGoalTemplate();
+
+/**
+ * @unreleased
+ */
+export default function Header() {
+    return (
+        <HeaderTemplate
+            Title={() => (form.settings?.showHeading ? <TitleTemplate text={form.settings.heading} /> : <></>)}
+            Description={() =>
+                form.settings?.showDescription ? <DescriptionTemplate text={form.settings.description} /> : <></>
+            }
+            Goal={() =>
+                form.goal?.show ? (
+                    <GoalTemplate
+                        currency={form.currency}
+                        type={form.goal.type as GoalType}
+                        goalLabel={form.goal.label}
+                        progressPercentage={form.goal.progressPercentage}
+                        currentAmount={form.goal.currentAmount}
+                        currentAmountFormatted={form.goal.currentAmountFormatted}
+                        targetAmount={form.goal.targetAmount}
+                        targetAmountFormatted={form.goal.targetAmountFormatted}
+                        totalRevenue={form.stats.totalRevenue}
+                        totalRevenueFormatted={form.stats.totalRevenueFormatted}
+                        totalNumberOfDonationsOrDonors={form.stats.totalNumberOfDonationsOrDonors}
+                        totalNumberOfDonationsOrDonorsLabel={form.stats.totalNumberOfDonationsOrDonorsLabel}
```

Comment:
This is the one case where something generic just kept confusing me which is why I made it so specific.  If the goal type is 'amount' it still displays the total number of donations.

Reply:
It just makes me nervous for future goals. For subscription based goals, for example, I think it would make sense for this to be a subscription count — and `totalNumberofDonationsOrDonorsOrSubscriptions` sounds... robust. 😂

Reply:
Yeah i'm fine changing it to `totalCountValue` for now - that's basically where I started.  I just changed it because I realized that unless it's the 'amount' goal this value is actually the same as `currentAmount` and kept confusing me.

Reply:
resolved e19f8641